app/bundles/CoreBundle/Assets/css/app/less/variables.less

```diff
@@ -2,6 +2,20 @@
 // Variables
 // --------------------------------------------------
 
+//== General
+//
+// Animations
+:root {
+    --timing:                                    .3s ease;
+    --all:                                       all var(--timing);
+}
+
+@media (prefers-reduced-motion: reduce) {
+    :root {
+        --timing:                                0;
+    }
+}
+
```

Comment:
Perhaps the reason for triplicatiion of this code in app.css, libraries.css and builder.css is that variables.less is imported into less files that are sources of all 3 css files? Is this the right place to add this place?

Reply:
Thanks for the discussion! I agree that it's not the goal of this PR to fix all LESS/CSS issues in the project. But it helped me to understand more.